addons/mail/static/src/components/chatter/chatter.xml

```diff
@@ -26,7 +26,7 @@
                     <t t-if="chatter.isAttachmentBoxVisible">
```

Comment:
remove `isAttachmentBoxVisible` and replace it by a condition on `chatter.attachmentBoxView`.

Indeed `attachmentBoxView` record should not exist if it is not displayed. This is a general principle for all view records.